cves/CVE-2002-0661.yml

```diff
@@ -41,19 +50,18 @@ bounty:
   announced:
```

Comment:
Was there no information online announcing the vulnerability before the fix was published?

Reply:
this is the announced line under the bounty section (which there was none), not the announced line for the vulnerability

Reply:
Totally correct, my bad!